htdocs/core/class/html.form.class.php

```diff
@@ -8040,7 +8040,6 @@ public function selectForForms($objectdesc, $htmlname, $preSelectedValue, $showe
 		if ($objectfield) {	// We must retreive the objectdesc from the field or extrafield
 			// Example: $objectfield = 'product:options_package'
 			$tmparray = explode(':', $objectfield);
-			$objectdesc = '';
```

Comment:
Why removing this line ?

Reply:
J'ai retiré cette ligne parce que la variable est défini dans le if et dans le else.
Mais c'est vrai qu'il y a un cas ou elle pourrait être redéfini si cette condition est false !empty($extrafields->attributes[$objectforfieldstmp->table_element]['param'][$key]['options']))

Je peux le remettre s'il y en besoin

Reply:
Hello @eldy
Awaiting your reply for this fix